ui/src/app/components/admin/job-templates/job-template-show/job-template-show.component.html

```diff
@@ -0,0 +1,142 @@
+<!--
+  ~ Copyright 2018 ABSA Group Limited
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div class="loading" *ngIf="loading">
+    <span class="spinner spinner-inverse">
+        Loading...
+    </span>
+</div>
+
+<div *ngIf="!loading && isSuccessfullyLoaded">
+  <form clrForm novalidate>
+    <div class="card">
+      <div class="card-block">
+        <div class="card-title notSelectable" (click)="toggleJobTemplateInfoAccordion()">
+          <clr-icon shape="angle" [dir]="isJobTemplateInfoHidden?'right':'down'"></clr-icon>
+          Job Template info
+        </div>
+        <div [hidden]="isJobTemplateInfoHidden" class="accordion-content">
+          <app-string-part
+            [isShow]="true"
+            [name]="'Job template name'"
+            [valueChanges]="changes"
+            [value]="jobTemplate.name"
+            [property]="'name'"
+            [partValidation]="partValidation">
+          </app-string-part>
+          <app-string-part
+            [isShow]="true"
+            [name]="'Form config'"
+            [valueChanges]="changes"
+            [value]="jobTemplate.formConfig"
+            [property]="'formConfig'"
+            [partValidation]="partValidation">
+          </app-string-part>
+          <app-string-part
+            [isShow]="true"
+            [name]="'Job type'"
+            [valueChanges]="changes"
+            [value]="jobTemplate.jobType.name"
+            [property]="'jobType.name'"
+            [partValidation]="partValidation">
+          </app-string-part>
+        </div>
+      </div>
+    </div>
+    <div class="card">
+      <div class="card-block">
+        <div class="card-title notSelectable" (click)="toggleJobTemplateAccordion()">
+          <clr-icon shape="angle" [dir]="isJobTemplateHidden?'right':'down'"></clr-icon>
+          Job Template
```

Comment:
How about "Job Template Parameters". Job template is actually the whole thing, so Job Template Info and this section together, so it could be a little confusing to name this section "Job template"

Reply:
Fixed, thanks